Tests/UnitTests/FoundationExtensions/DecoderExtensionTests.swift

```diff
@@ -89,7 +89,7 @@ class DecoderExtensionsIgnoreErrorsTests: TestCase {
     }
 
     func testIgnoresErrors() throws {
-        let json = "{\"url\": \"not a! valid url@\"}"
```

Comment:
Hmm so it accepts this as a URL? 🤯 

I wonder if we should add back this test (passing) so we get notified if they revert this change in future betas...

Reply:
I mean it doesn't really matter 😅 these tests were just to verify that we handle parsing errors, which we're still testing.